app/models/setting/general.rb

```diff
@@ -28,4 +28,10 @@ def self.load_defaults
 
     true
   end
+
+  def self.validate_settings(errors)
+    ["idle_timeout", "proxy_request_timeout", "entries_per_page"].each do |s|
```

Comment:
Yeah, I will remove and try to do per-entry approach. But I will keep `entries_per_page` as a harmless example for the general settings just to have something in the file.